sql/4_2_1-to-4_2_2_upgrade.sql

```diff
@@ -160,3 +160,10 @@ INSERT INTO list_options (list_id, option_id,title, seq) VALUES ('issue_subtypes
 UPDATE `clinical_rules` SET `amc_2014_stage1_flag` = 1, `amc_2014_stage2_flag` = 1 WHERE `id` = 'med_reconc_amc' AND `pid` = 0;
 UPDATE `clinical_rules` SET `amc_2014_stage1_flag` = 1, `amc_2014_stage2_flag` = 1 WHERE `id` = 'med_reconc_amc' AND `pid` = 0;
 
+#IfNotRow globals gl_name first_day_week
+INSERT INTO globals ( gl_name, gl_index, gl_value ) VALUES ( 'first_day_week', 0, '1' );
+#EndIf
```

Comment:
This is not necessary. During upgrade(and patches and whenever save on Administration->Globals) the new globals are brought in via library/globals.inc.php .